### PR TITLE
dynamic module for workallocation

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/work-allocation/work-allocation-feature.routes.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      path: 'task-manager',
+      component: TaskManagerComponent,
+      canActivate: [ HealthCheckGuard ],
+      data: {
+        title: 'HMCTS Manage WorkAllocation | Task Manager'
+      }
+    }
+   ];
+
+export const casesRouting: ModuleWithProviders = RouterModule.forChild(ROUTES);
```
Comment (paul-graham) [inline src/work-allocation/work-allocation-feature.routes.ts:26]: I think this should be "workAllocationRouting".

File: src/work-allocation/work-allocation.module.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { NgModule } from '@angular/core';
+import { HttpModule } from '@angular/http';
+import { MatDialogModule } from '@angular/material';
+import {SharedModule} from '../app/shared/shared.module';
+// from containers
+import * as fromContainers from './containers';
+import { casesRouting } from './work-allocation-feature.routes';
```
Comment (paul-graham) [inline src/work-allocation/work-allocation.module.ts:9]: See above about the naming.

**Review by paul-graham — CHANGES_REQUESTED** [review]
`casesRouting` should be changed to `workAllocationRouting`.

File: src/work-allocation/work-allocation.module.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/work-allocation/work-allocation.module.ts]: Done

File: src/work-allocation/work-allocation-feature.routes.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/work-allocation/work-allocation-feature.routes.ts]: Done

**Review by paul-graham — APPROVED** [review]
Looks good to me now.

**Pull request merged**